xapian-core/api/valuerangeproc.cc

```diff
@@ -451,23 +451,23 @@ static const int size_map[26] = {
     0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
 };
 
-static int
-get_bytes(char ch)
+static unsigned int
+get_size_multiplier_from_suffix(const char ch)
 {
 	// It takes character as input and return number of bytes
 	// B - 1 byte
 	// K - 1024 bytes
 	// M - 1024*1024 bytes
 	// G - 1024*1024*1024 bytes
-	if (ch >= 'A' && ch <= 'Z' && size_map[ch - 'A'] > 0) {
-    return size_map[ch - 'A'];
+	if (ch >= 'A' && ch <= 'Z') {
+		return size_map[ch - 'A'];
 	} else {
-	return 0;
+		return 0;
 	}
```

Comment:
These two return statements seem to be over-indented - each indent level should be 4 columns, but the indent here is two tabs (16 columns) but the `if` is indented by one tab (8 columns).